formats/src/main/scala/com/gu/scanamo/DynamoFormat.scala

```diff
@@ -310,7 +311,14 @@ object DynamoFormat extends EnumDynamoFormat {
     )(javaListFormat)
 
   private val javaNumSetFormat = attribute(_.getNS, "NS")(_.withNS)
-  private val javaStringSetFormat = attribute(_.getSS, "SS")(_.withSS)
+  private val javaStringSetFormat: DynamoFormat[java.util.List[String]] =
```

Comment:
Not too keen on having instances for mutable types, but I see that was the direction chosen previously.